### PR TITLE
Remove mac os installers etc

### PR DESCRIPTION
**Pull request by ascjones**
Part of #8649. Reviewers please confirm I have removed all the correct artifacts.

@5chdn can you set up a test release to make sure the pipelines still work?

### CODE REVIEW

**Comment by 5chdn** [comment]
* [new tag]             v1.12.0-ci4 -> v1.12.0-ci4


https://gitlab.parity.io/parity/parity/pipelines/20746

File: scripts/gitlab-build.sh [hunk 1]
```diff
@@ -291,11 +275,9 @@ case $BUILD_PLATFORM in
   x86_64-apple-darwin)
     STRIP_BIN="strip"
     PLATFORM="x86_64-apple-darwin"
-    EXT="pkg"
```
Comment (5chdn) [inline scripts/gitlab-build.sh]: Wondering where this environment variable is used.

Comment (dvdplm) [reply scripts/gitlab-build.sh]: @5chdn looks like it was used to build up a filename [here](https://github.com/paritytech/parity/pull/8875/files#diff-3e79197c2ea5f6f50208567f975976a4L162) and afaict that's the only place.

Comment (ascjones) [reply scripts/gitlab-build.sh]: Yep looks like it. Perhaps we can remove `EXT` for the other platforms?

**Comment by 5chdn** [comment]
```❯ aws s3 ls s3://builds-parity/v1.12.0-ci4/x86_64-apple-darwin/
2018-06-12 20:16:18    2034532 ethkey
2018-06-12 20:16:23         76 ethkey.md5
2018-06-12 20:16:25        108 ethkey.sha256
2018-06-12 20:16:08    2018260 ethstore
2018-06-12 20:16:12         78 ethstore.md5
2018-06-12 20:16:16        110 ethstore.sha256
2018-06-12 20:15:43   33027096 parity
2018-06-12 20:15:57    6960744 parity-evm
2018-06-12 20:16:03         80 parity-evm.md5
2018-06-12 20:16:06        112 parity-evm.sha256
2018-06-12 20:15:53         76 parity.md5
2018-06-12 20:15:55        108 parity.sha256
2018-06-12 20:16:28    3844860 whisper
2018-06-12 20:16:33         77 whisper.md5
2018-06-12 20:16:35        109 whisper.sha256
```

**Pull request merged**